### DISCUSSION TITLE
Unable to view .ipynb file

### DISCUSSION THREAD

**Discussion by llmzyj** [community/community#62215] [Mobile] [answered]
Unable to view .ipynb file on Github Mobile(V1.123.1). I hope this function can be achieved. Thanks!

**Comment by mxie** [comment, marked as answer]
Thanks for reaching out to us @Limnium! Unfortunately, support for Jupyter files is not currently on our roadmap. However, if we see enough demand for it, we may revisit this idea.

**Reply by AumGupta** [reply]
I feel this feature is much needed. I write a lot of my class notes on .ipynb and many a times I feel the need to revise on my phone. Please try to incorporate this.

**Comment by SAMIULLAH-KALHORO-DEV** [comment]
The **.ipynb** file extension is associated with Jupyter Notebook files, which are interactive documents commonly used for data analysis, code execution, and visualizations. To view .ipynb files, you need to have a Jupyter Notebook environment

### Here are some options to view .ipynb files:

**Jupyter Notebook or JupyterLab**
**Jupyter Online Viewer:**
**nbviewer:**
**VSCode:**

**Comment by sgindeed** [comment]
For accessing ipynb files you can also use the VS code. If you upload the files in a Github repository it will stored as a Jupyter Notebook file. If you have stats showing on your github profile readme section then it'll also be shown there as Jupyter Notebook file.

**Comment by MangalAdhav** [comment]
till now it was supporting and opening , now suddenly .ipynb is not opening

**Comment by edenst24** [comment]
[[]()]()

**Comment by edenst24** [comment]
Thanks for reaching out to us @Limnium! Unfortunately, support for Jupyter files is not currently on our roadmap. However, if we see enough demand for it, we may revisit this idea.

**Comment by edenst24** [comment]
> Thanks for reaching out to us @Limnium! Unfortunately, support for Jupyter files is not currently on our roadmap. However, if we see enough demand for it, we may revisit this idea.

**Comment by AumGupta** [comment]
> Unable to view .ipynb file on Github Mobile(V1.123.1). I hope this function can be achieved. Thanks!

@lichium have you figured out some alternative? If yes, please let me know.

**Comment by Jan-Dyndor** [comment]
I know a lot of time have passed but i found one way - `reviewnb`. Check it out 
LINK : https://www.reviewnb.com/ 
Their GH: https://github.com/ReviewNB

**Comment by LavanyaURao** [comment]
Hellow, im facing a problem in preview of ipnby file. any of u guys facing the same?

<img width="1469" height="791" alt="image" src="https://github.com/user-attachments/assets/63eed292-10ea-472d-9de8-e25fb1113b9b" />

**Reply by seohyunjun** [reply]
How is it going.. 

Github Apps don't support viewing *.ipynb file.

**Reply by LavanyaURao** [reply]
no before we were able to, now its showing some error
